services/venice-controller/src/main/java/com/linkedin/venice/controller/logcompaction/NominationFilter.java

```diff
@@ -0,0 +1,12 @@
+package com.linkedin.venice.controller.logcompaction;
+
+import com.linkedin.venice.meta.StoreInfo;
+
+
+public interface NominationFilter {
```

Comment:
Add java doc.
Suggest renaming these to `CandidateFilter` to make it more explicit. `Nomination` and `Filter` are both actions and it is not evident on what entity the actions are applied on from the name.